src/SymbolCollector.Console/Program.cs

```diff
@@ -282,7 +292,7 @@ internal class Args
         public string? Path { get; }
         public string? Symsorter { get; }
         public string? BundleId { get; }
-        public string? BatchType { get; }
+        public BatchType? BatchType { get; }
```

Comment:
Will this work? Since the library binding arguments might not support binding to the enum